### ISSUE TITLE
Pre tag makes page overflow on the right

### ISSUE THREAD

**Issue by Smrman** [InternationalScratchWiki/ScratchWikiSkin2#42]
Hi,

On a small screen, when there is a `<pre>` tag in the page, it sometimes overflows on the right side, leaving a huge scrollable blank space.

_See below (Google Chrome mobile device emulator)_
![capture](https://user-images.githubusercontent.com/17651422/45723031-98153e00-bbaf-11e8-8717-a75e400be1ab.PNG)

Could be solved by adding scroll when it overflows.
Just one line of CSS is needed and it works...
```css
pre {
overflow-x: scroll;
}
```

**Comment by Kenny2github** [comment]
What page is this? I want to try to reproduce this for myself.

**Comment by apple502j** [comment]
@Kenny2github It seems like frwiki talk.
https://fr.scratch-wiki.info/wiki/Discussion_Scratch_Wiki_en_Fran%C3%A7ais:Accueil_de_la_Communaut%C3%A9#Wiki Wednesday

Full size full screen, overflown for me, Firefox 62.
```css
#content pre {
    white-space: pre;
}
```
disables auto-wrapping.
Maybe `pre-wrap`?

**Comment by Kenny2github** [comment]
Fixed in ed049ab
